openslides/utils/auth.py

```diff
@@ -75,7 +88,10 @@ def user_to_collection_user(user: AnyUser) -> Optional[CollectionElement]:
             "Unsupported type for user. Only CollectionElements for users can be"
             "used. Not {}".format(user.collection_string))
     elif isinstance(user, int):
-        user = CollectionElement.from_values(User.get_collection_string(), user)
+        if user == 0:
```

Comment:
In which case may user be 0?

Reply:
In some cases, we use 0 for anonymous user. See for example https://github.com/OpenSlides/OpenSlides/blob/32e20aa6708b14f42fbcfcaa1e0792cb20af514b/openslides/utils/autoupdate.py#L72 in the current master. I don't think, that there is any case in this branch, where this line is needed. If you like, I can remove it.